app/controllers/api/v1/items/find_items_controller.rb

```diff
@@ -14,4 +15,10 @@ def item_params
     params.permit(:id, :description, :name, :unit_price, :merchant_id, :created_at, :updated_at)
   end
 
+  def format_unit_price
```

Comment:
@s-espinosa does it make more sense to do this here, or to seed the data in the correct format? Thanks!